project/ApiDefinitionsGenerator.scala

```diff
@@ -0,0 +1,164 @@
+
+import scala.collection.mutable.ArrayBuffer
+import scala.util.parsing.json.JSON
+import scala.io.Source
+object ApiDefinitionsGenerator {
+
+  def modules(): Seq[module] = {
+    //todo: read term_info.json from java driver
+    val jsonFilename = "term_info.json"
+    var modules = ArrayBuffer[module]()
```

Comment:
What is the right way to parse couple of objects? I created mutable collection for adding modules one by one :)